docs/specifications/tests/DNSSEC-TP/dnssec01.md

```diff
@@ -33,87 +49,162 @@ Algorithm number | Algorithm (or description)
 4                | SHA-384
 5-255            | (Unassigned)
 
+This test case will verify that the Zonemaster implementation has support for the
+DS digest algorithm of the DS record found, and if not output a message tag. If
+the support is missing other test cases will not be able to verify that DS
+record.
+
+
+## Scope
+
+This test case will query the name servers of the parent zone, and will just
+ignore non-responsive name servers or name servers not giving a correct DNS
+response for an authoritative name server.
+
+If no DS record is found in the parent zone then this test case will be
+terminated without outputting any message tag.
+
+This test case does not report if the parent servers give inconsistent responses.
+
+If the *Child Zone* is the root zone, then it has no parent zone, and no DS
+records cannot be fetch.
```

Comment:
double negation: "no DS records cannot be fetch", would it be possible to rephrase?